### PR TITLE
Update translations from Weblate

### PR DESCRIPTION
**Pull request by zetok**
This change is [Reviewable](https://reviewable.io/reviews/qtox/qtox/4168)

### CODE REVIEW

File: src/widget/form/settings/generalform.cpp [hunk 1]
```diff
@@ -104,7 +106,9 @@ static QStringList langs = {"Arabic",
                             "Türkçe",
                             "ئۇيغۇرچە",
                             "Українська",
-                            "中文（中国）"};
+                            "中文（中国）",
+                            "Chinese (Taiwan)"
```
Comment (zetok) [inline src/widget/form/settings/generalform.cpp:110]: @LNDDYL I don't know what should be here. Or whether it's in the right position on the list. Help?

**Comment by Diadlo** [comment]
Reviewed 40 of 40 files at r1.
Review status: all files reviewed at latest revision, 1 unresolved discussion.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/qtox/qtox/4168)*

File: src/widget/form/settings/generalform.cpp (hunk 1 above)
Comment (ghost) [reply src/widget/form/settings/generalform.cpp:110]: @zetok
"Chinese (Taiwan)" is "中文（台灣）".

Comment (ghost) [reply src/widget/form/settings/generalform.cpp:110]: What you did is right, but you should use Chinese.

**Comment by zetok** [comment]
Deprecated by #4190.

**Pull request closed without merging**